add Webpack Butternut Plugin (modern minifier)

MaxGraey opened the pull request.

MaxGraey committed:
add Webpack Butternut Plugin (modern minifier)

evilebottnawi commented:
@MaxGraey https://github.com/webpack-contrib/awesome-webpack/pull/69#issuecomment-302417959

apercu committed:
feat(plugins): add prepack plugin (#68)

joshwiens committed:
style: fix line wrap

mastilver committed:
feat: add modules-cdn-webpack-plugin (#71)

lostpebble committed:
docs(README): add `generate-package-json-webpack-plugin` (#72)

arthurbergmz committed:
docs(README): add `webpack-pwa-manifest` plugin (#66)

* Added webpack-pwa-manifest plugin

jstcki committed:
feat: Add responsive-loader (#73)

dhavalchaudhary committed:
feat: Add Webpack Academy in courses. (#74)

* Added a learning resource

* Removed trailing slash

* Removed the "by" section.

styopdev committed:
feat: Added link to interview questions repo (#75)

Commit:
Update Sean Larkin Bio Information (#84)

@TheLarkInn Feel free to add another PR to fix if it isn't the way you want it.

Commit:
feat: Add Polymer loader (#85)

Commit:
fix: correct broken links (#83)

gregnb committed:
docs(README): add `filemanager-webpack-plugin` (#78)

grgur committed:
feat: Added the Budgeting sample application (#81)

This sample app was started as a collection of experiences developing production apps such as uniqlo.com. It's been actively maintained, upgraded, and enjoyed by the community.

darrenscerri committed:
feat: Add Duplicate Package Checker Webpack Plugin (#88)

Warns you when multiple versions of the same package exist in your bundle

https://github.com/darrenscerri/duplicate-package-checker-webpack-plugin

Commit:
feat: Add imagemin-webpack plugin/loader (#87)

Much more recent than [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader) and [imagemin-webpack-plugin](https://github.com/Klathmon/imagemin-webpack-plugin) and boasts:

* Granular dependency installing (good for CI + bloat)
* Both plugin + loader to support multiple spots for using images
* Company supported vs individual

milewski committed:
feat: Added bin-exec-loader (#62)

mrsteele committed:
feat: Adds dotenv-webpack (#91)

radum committed:
docs: added friendly-errors-webpack-plugin to the plugins list (#80)

* Added friendly-errors-webpack-plugin to the plugins list

* Shortened description for Friendly errors plugin

stephen committed:
docs: add graphql-loader (#92)

rhys-vdw committed:
docs: add fengari-loader to list of loaders (#100)

stephencookdev committed:
docs: add 'Speed Measure Plugin' to the list of Webpack Tools (#97)

evilebottnawi commented:
@MaxGraey Please rebase

z-ZYS-s committed:
docs(loaders): add icon-font-loader (another loader for icon font) (#86)

Ffloriel committed:
docs(plugins): add purgecss webpack plugin (#93)

* add purgecss webpack plugin

* change description

MaxGraey committed:
Merge branch 'master' into master

MaxGraey commented:
Done!
